### PR TITLE
CLDR-15141 add rohingya lc

### PR DESCRIPTION
**Pull request by stenshamn**
CLDR-15141

- [X] This PR completes the ticket.

### CODE REVIEW

**Review by rickmcgowan — DISMISSED** [review]

**Comment by srl295** [comment]
@stenshamn the commit message and PR needs to start with `CLDR-15141` … no space after CLDR.. I fixed the PR but cant fix the commit

**Review by srl295 — DISMISSED** [review]

File: tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/Locales.txt [hunk 1]
```diff
@@ -886,6 +886,8 @@ Mozilla	;	und	;	modern
 
 Rodakych ; pcm ; modern ; Nigerian Pidgin
 
+rohingyazuban	;	rhg	;	modern
```
Comment (srl295) [inline tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/Locales.txt:889]: LGTM but do we need to have the script? are they covering rhg_Arab and rhg_Rohg ?

Comment (stenshamn) [reply tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/Locales.txt:889]: I figured by leaving it unstated they would be able to work with both?

**Comment by stenshamn** [comment]
> @stenshamn the commit message and PR needs to start with `CLDR-15141` … no space after CLDR.. I fixed the PR but cant fix the commit

I caught that, but thought I fixed it in when I squashed the commits. That didn't stick?

File: tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/Locales.txt (hunk 1 above)
Comment (pedberg-icu) [reply tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/Locales.txt:889]: The default script is Rohg, so unspecified implies that I think.

**Review by pedberg-icu — APPROVED** [review]

**Pull request merged**